backend/src/Command/Dev/DevSeedCommand.php

```diff
@@ -0,0 +1,119 @@
+<?php 
+
+namespace App\Command\Dev;
+
+use App\Factory\CollectionFactory;
+use App\Factory\ItemFactory;
+use App\Factory\PublicationFactory;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\HttpKernel\KernelInterface;
+
+/**
+ * @codeCoverageIgnore
+ */
+#[AsCommand(
+    name: 'app:dev:seed',
+    description: 'Seeds the database with test data for development purposes.'
+)]
+class DevSeedCommand extends Command
+{
+    public function __construct(
+        private KernelInterface $kernel,
+    ) {
+        parent::__construct();
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $env = $this->kernel->getEnvironment();
```

Comment:
- use faker for data
- add a collection, then 2 publications for that collections, then multiple items